### PR TITLE
Replace Plus API with People API due Plus shutdown

### PR DESCRIPTION
**Pull request by drfenixion**
Needs to activate Google People API (in Google console)

### CODE REVIEW

File: src/Google-Plus/Provider.php [hunk 1]
```diff
@@ -51,10 +50,11 @@ protected function getTokenUrl()
     protected function getUserByToken($token)
     {
         $response = $this->getHttpClient()->get(
-            'https://www.googleapis.com/plus/v1/people/me', [
+            'https://people.googleapis.com/v1/people/me', [
             'headers' => [
                 'Authorization' => 'Bearer '.$token,
             ],
+            'query'   => array('personFields' => 'emailAddresses,names,photos'),
```
Comment (m1guelpf) [inline src/Google-Plus/Provider.php]: Please use the new array style

File: src/Google-Plus/Provider.php [hunk 2]
```diff
@@ -66,10 +66,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'     => $user['id'], 'nickname' => Arr::get($user, 'nickname'),
-            'name'   => $user['displayName'],
-            'email'  => $user['emails'][0]['value'],
-            'avatar' => Arr::get($user, 'image.url'),
+            'id' => $user['names'][0]['metadata']['source']['id'],
+            'nickname' => $user['names'][0]['displayName'],
```
Comment (m1guelpf) [inline src/Google-Plus/Provider.php]: Are all of this values always set?

File: src/Google-Plus/Provider.php [hunk 3]
```diff
@@ -66,10 +66,11 @@ protected function getUserByToken($token)
...
+            'name' => $user['names'][0]['displayName'],
+            'email' => $user['emailAddresses'][0]['value'],
+            'avatar' => $user['photos'][0]['url'],
```
Comment (m1guelpf) [inline src/Google-Plus/Provider.php]: Is the first photo always the avatar? What happens if the user has no avatar?

Comment (drfenixion) [reply src/Google-Plus/Provider.php]: If not set any ava Google uses an image with your name first char as the avatar. I removed my ava and  Google automatically set this ava for me https://lh6.googleusercontent.com/-IPi1e97NLh4/AAAAAAAAAAI/AAAAAAAAAAA/ACevoQNXgMp-CZ4U_4r_aIDPCcDJil4N_g/s100/photo.jpg

File: src/Google-Plus/Provider.php (hunk 2 above)
Comment (drfenixion) [reply src/Google-Plus/Provider.php]: Not found info about it. Due to this add some checks via Arr:get

Docs and test tool
https://developers.google.com/people/api/rest/v1/people/get?apix_params=%7B%22resourceName%22%3A%22people%2Fme%22%2C%22personFields%22%3A%22photos%22%7D

File: src/Google-Plus/Provider.php (hunk 1 above)
Comment (drfenixion) [reply src/Google-Plus/Provider.php]: Done

**Pull request merged**